### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by Riki7745** [emoacht/Monitorian#430] [Q&A]
Hi, I'd like to translate it but don't know how to, any help ? thanx in advance.

**Comment by emoacht** [comment]
Please read [Globalization](https://github.com/emoacht/Monitorian#globalization) section in readme.
You can find other languages' examples in https://github.com/emoacht/Monitorian/tree/master/Source/Monitorian.Core/Properties

**Comment by Riki7745** [comment]
I'll give it a try right now ! Thanx mate.

**Reply by Riki7745** [reply]
I opened the "Resources.fr.resx" (with Notepads)...but I don't know what to translate, because it's a huge file, and I don't want to translate what doesn't need to...

Edit : I opened it in GitHub and I saw what to translate, but few things missing (like "Use accent color for brightness", "Order by monitors arrangement"...) in the menu, that can be translated but doesn't appear translated...
![capture](https://user-images.githubusercontent.com/377289/226171955-44c115b4-a010-492a-9e8f-1546b795f9fa.jpg)
